mcr.cpp

```diff
@@ -15,19 +19,35 @@ bool isWin(char game[3][3]){
 
 int main(){
     int i, j;
-    char game[3][3] = {' '}; // Tic-tac-toe
-    char player1 = 'X';
-    char player2 = 'O';
+    i = -1;
+    j = -1;
+    char game[3][3] = {'0'}; // Tic-tac-toe
+    char c;
     bool turn = true; // false for player 1's turn, true for player 2's turn. Player 1 first.
     cout << "X = Player 1" << endl << "O = Player 2" << endl;
     for (int n=0; n<9; n++){
         turn = !turn;  // use the not-operator to change true to false or false to true.
-        if (turn == false)
+        if (turn == false){
             cout << "Player 1: ";
-        else
+        }
+        else{
             cout << "Player 2: ";
-        cout << "Which cell to mark? i:[1..3], j:[1..3]: ";
+        }
+
+        rewrite:
+        cout << "Which cell to mark? i:[0..2], j:[0..2]: ";
         cin >> i >> j;
+
+        if(game[i][j] != 'X' && game[i][j] != 'O' && i <= 2 && i >= 0 && j <= 2 && j >= 0) { //check if the input is appropriate.
```

Comment:
Still need to determine if the input is not a number and output a hint

Reply:
I really have no idea with this, we just hope that no one will use characters.